_includes/js/objects.md

```diff
@@ -172,12 +172,13 @@ var playerName = gameScore.get("playerName");
 var cheatMode = gameScore.get("cheatMode");
 </code></pre>
 
-The three special reserved values are provided as properties and cannot be retrieved using the 'get' method nor modified with the 'set' method:
+The four special reserved values are provided as properties and cannot be retrieved using the 'get' method nor modified with the 'set' method:
 
 <pre><code class="javascript">
 var objectId = gameScore.id;
 var updatedAt = gameScore.updatedAt;
 var createdAt = gameScore.createdAt;
+var acl = gamescore.ACL;
```

Comment:
Typo. Should be **gameScore**.

Also, ACL is not a valid property according to docs (and source code). It should be **getACL()**. 

See here: https://github.com/parse-community/Parse-SDK-JS/blob/master/src/ParseObject.js#L917